jaeger-dropwizard/README.md

```diff
@@ -0,0 +1,91 @@
+# Dropwizard #
+
+## Maven Dependency ##
+If you are using dropwizard then you can add the following:
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-dropwizard</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
+## Instrumentation ##
+This project includes instrumentation for everything that implements the JAX-RS 2.0
```

Comment:
"for everything that implements" is a strong statement - all we have are HTTP filters